src/qibolab/serialize.py

```diff
@@ -63,9 +90,17 @@ def dump_qubits(qubits: QubitMap, pairs: QubitPairMap) -> dict:
         if len(natives) > 0:
             native_gates["two_qubit"][f"{p[0]}-{p[1]}"] = natives
     # add qubit characterization section
+    characterization = {
+        "single_qubit": {q: qubit.characterization for q, qubit in qubits.items()},
+        "coupler": {},
+    }
+    if couplers:
+        for coupler in couplers.values():
+            characterization["coupler"][coupler.name] = 1
```

Comment:
I would prefer
```suggestion
    characterization = {
        "single_qubit": {q: qubit.characterization for q, qubit in qubits.items()},
    }
    if couplers is not None:
        characterization["coupler"] = {coupler.name: 1 for coupler in couplers.values()}
```
so that the `coupler:` section is only created when couplers exist. Otherwise we will create an empty section for runcards that don't have couplers.

Also, why this is set as 1? I believe you actually need something like
```suggestion
    characterization = {
        "single_qubit": {q: qubit.characterization for q, qubit in qubits.items()},
    }
    if couplers is not None:
        characterization["coupler"] = {c.name: {"sweetspot": c.sweetspot} for c in couplers.values()}
```
to properly dump the sweetspot as in the original Zurich runcard.